src/main/java/shukaro/warptheory/items/itemCleanserMinor.java

```diff
@@ -0,0 +1,122 @@
+package shukaro.warptheory.items;
+
+import cpw.mods.fml.relauncher.Side;
+import cpw.mods.fml.relauncher.SideOnly;
+import net.minecraft.client.renderer.texture.IIconRegister;
+import net.minecraft.creativetab.CreativeTabs;
+import net.minecraft.entity.player.EntityPlayer;
+import net.minecraft.item.EnumAction;
+import net.minecraft.item.EnumRarity;
+import net.minecraft.item.Item;
+import net.minecraft.item.ItemStack;
+import net.minecraft.util.IIcon;
+import net.minecraft.util.StatCollector;
+import net.minecraft.world.World;
+import shukaro.warptheory.WarpTheory;
+import shukaro.warptheory.handlers.WarpHandler;
+import shukaro.warptheory.util.ChatHelper;
+import shukaro.warptheory.util.Constants;
+import shukaro.warptheory.util.FormatCodes;
+
+import java.util.List;
+import java.util.Locale;
+
+public class ItemCleanserMinor extends Item
```

Comment:
Why not just extend from ItemCleanser and override the methods that you change? I"m assuming most of them are going to be pretty similar.